sky/skylet/attempt_skylet.py

```diff
@@ -27,7 +27,8 @@ def restart_skylet():
 
 
 proc = subprocess.run(
-    'ps aux | grep -v "grep" | grep "sky.skylet.skylet" | grep "python3 -m"',
+    f'ps aux | grep -v "grep" | grep "sky.skylet.skylet" | '
+    f'grep "{constants.SKY_PYTHON_CMD} -m"',
```

Comment:
Why not grep this too on L16?

Reply:
L16 is for backward compatibility, as we need to make sure that the skylet started before this PR (will not have the full path to the python executable) will be correctly killed.